Makefile

```diff
@@ -174,6 +174,14 @@ rpc:
 	@$(call print, "Compiling protos.")
 	cd ./lnrpc; ./gen_protos.sh
 
+rpc-format:
+	@$(call print, "Formatting protos.")
+	cd ./lnrpc; clang-format -i *.proto **/*.proto
+
+rpc-check: rpc-format rpc
+	@$(call print, "Verifying protos.")
+	if test -n "$$(git describe --dirty | grep dirty)"; then echo "Protos not properly formatted or not compiled with v3.4.0"; exit 1; fi
```

Comment:
Nice way to ensure formatting :slightly_smiling_face:

Reply:
Yes, I like it too! It's @cfromknecht's idea, btw.